### PR TITLE
Getting users from a server

### PR DESCRIPTION
**Pull request by mpdavis**
I needed this code in order to keep working on the scanner.  You are welcome to look it over, but this all works correctly.

Please review:
@qinxiaoapp
@computmaxer

### CODE REVIEW

File: server/models/server.py [hunk 1]
```diff
@@ -21,6 +21,31 @@ class Server(Document, AuditableMixin):
...
+        """
+        users = set()
+
+        # Getting users added directly to the server
+        for user in self.user_list:
+            users.append(user)
+
+        # Getting users that are in groups on the server
+        for group in self.group_list:
+            for user in group:
```
Comment (computmaxer) [inline server/models/server.py]: I think this should be `for user in group.user_list:`

File: server/models/server.py [hunk 2]
```diff
@@ -21,6 +21,31 @@ class Server(Document, AuditableMixin):
...
+    def get_users(self):
+        """
+        Gets all of the users that have access to the machine.
+        This includes users added directly and through groups.
+        """
+        users = set()
+
+        # Getting users added directly to the server
+        for user in self.user_list:
+            users.append(user)
```
Comment (computmaxer) [inline server/models/server.py]: set uses `.add()` not `.append()`

File: server/models/server.py [hunk 3]
```diff
@@ -21,6 +21,31 @@ class Server(Document, AuditableMixin):
...
+
+        # Getting users that are in groups on the server
+        for group in self.group_list:
+            for user in group:
+                users.append(user)
```
Comment (computmaxer) [inline server/models/server.py]: set uses `.add()` not `.append()`

**Comment by mpdavis** [comment]
@computmaxer: updated

**Comment by computmaxer** [comment]
:+1:

**Comment by qinxiaoapp** [comment]
:+1:

**Comment by qinxiaoapp** [comment]
:+1:

**Pull request merged**